src/ai/backend/client/cli/admin/user.py

```diff
@@ -219,6 +230,7 @@ def add(
     need_password_change,
     allowed_ip,
     description,
+    enable_sudo_session,
```

Comment:
Column name is to represent the state, not the action. Please change the name.
```suggestion
    sudo_session_enabled,
```